templates/flashes_helper.rb

```diff
@@ -0,0 +1,5 @@
+module FlashesHelper
+  def user_facing_flashes
+    flash.to_hash.slice("alert", "error", "notice", "success")
```

Comment:
Yes. That's what I meant. Shame it doesn't exist but now I know what my
next rails pr is...

On Friday, February 20, 2015, Tute Costa <notifications@github.com> wrote:

> In templates/flashes_helper.rb
> <https://github.com/thoughtbot/suspenders/pull/507#discussion_r25089292>:
>
> > @@ -0,0 +1,5 @@
> > +module FlashesHelper
> > +  def user_facing_flashes
> > +    flash.to_hash.slice("alert", "error", "notice", "success")
>
> @derekprior <https://github.com/derekprior> do you mean grab the list of
> user facing flash keys from Rails? I couldn't find in
> https://github.com/rails/rails/blob/master/actionpack/lib/action_dispatch/middleware/flash.rb
> a way to do that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/507/files#r25089292>.
>